frontend/Controls/NavigateButtons

```diff
@@ -0,0 +1,11 @@
+import QtQuick 2.0
+import QtQuick.Controls 2.2
+
+ButtonModal {
```

Comment:
@Poseyy It might be wise to reconsider adding this control `NavigateButtons` since it isn't really any different than the control it's based on, `ButtonModal`. I also noticed that this control adds letter spacing, which isn't consistent with our other buttons - is that intentional?